pkg/controllers/provisioning/scheduling/preferences.go

```diff
@@ -35,6 +35,18 @@ type Preferences struct {
 	ToleratePreferNoSchedule bool
 }
 
+// RelaxTopologyPreferences relaxes preferred constraints for TSC, pod affinity, and pod anti-affinity.
+func (p *Preferences) RelaxPreferrerdTopologyConstraints(ctx context.Context, pod *v1.Pod) {
```

Comment:
I get that we call this "topology" internally, but kinda weird to name it this -- is a more appropriate name `RelaxPreferredTopologyAndPodAffinityConstraints`?

Reply:
Yeah, I can go either way. I've internalized the concept that these are all "topology" constraints so this felt natural, but I'm happy to update it.